Doc/license.rst

```diff
@@ -1132,3 +1132,39 @@ The file is distributed under the 2-Clause BSD License::
   THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
   (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
   THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+Zstandard bindings
+------------------
+
+Zstandard bindings in :file:`Modules/_zstd` and :file:`Lib/compression/zstd`
+are based on code from the pyzstd library, copyright Ma Lin and contributors.
+The code is distributed under the 3-Clause BSD License::
```

Comment:
Minor, but to because this isn't just a copy of the pyzstd work, we should be clear that it is only the original upstream that is BSD-3.
```suggestion
The ``pyzstd`` code is distributed under the 3-Clause BSD License::
```